packages/react-components/src/components/MessageThread.tsx

```diff
@@ -1127,6 +1137,8 @@ export const MessageThreadWrapper = (props: MessageThreadProps): JSX.Element =>
                   mentionOptions={mentionOptions}
                   /* @conditional-compile-remove(attachment-download) @conditional-compile-remove(attachment-upload) */
                   onRenderAttachmentDownloads={onRenderAttachmentDownloads}
+                  /* @conditional-compile-remove(rich-text-editor) */
+                  enableRichTextEditor={enableRichTextEditor}
```

Comment:
Just wanted to clarify if attachments will be added to RTE edit box in a separate PR